### DISCUSSION TITLE
Pi4 notary journal

### DISCUSSION THREAD

**Discussion by jayket** [codenotary/immudb#1574] [Q&A] [answered]
I came across this when I was looking for a way to build and selfhost a electronic notary journal. I liked it because the data can be verified and not deleted. My question is will this run on a raspberry pi 4 armv7? Also do you think it would work with something like budibase as the front-end? I’m new to databases so any help would be greatly appreciated. Thanks in advance.

**Comment by Razikus** [comment, marked as answer]
hello!

Here is an example on raspberry pi 64 bit usage (command uname -a to check if architecture is aarch64)
https://asciinema.org/a/9xxBDDPjhel5OIP8hgwiQX4GA

yes immudb will work on armv7
but you will need to build yourself
there is another record with fresh installation of 32bit Raspberry, you can just follow all steps
https://asciinema.org/a/HgqB8Ok3GP7euSyak9mVFd05q

**Reply by jayket** [reply]
Thank you for the videos they will definitely help especially the armv7 one. One last question have you come across this running in docker for the armv7 platform? Just easer to manage things with portioner in that capacity. Thanks again

**Reply by Razikus** [reply]
yes,
probably just as simple as executing
```
curl https://get.docker.com | bash
```

be aware as not all the images are architecture aware unfortunately
Go builder images should be

**Reply by Razikus** [reply]
quickly check on RPI armv7 - immudb is building with available Dockerfile, so just
```
docker build -t immudb .
```
